LibOS/shim/src/bookkeep/shim_vma.c

```diff
@@ -661,7 +684,6 @@ static int __bkeep_mprotect (struct shim_vma * prev,
         __insert_vma(new, prev);
         assert(!prev || prev->end <= new->end);
         assert(new->start < new->end);
-        assert(new->end <= cur->start);
```

Comment:
Any reason for removing this assert?

Reply:
Because it is untrue. "new" is not necessarily the previous node of "cur".